contracts/Auctioneer.sol

```diff
@@ -125,19 +125,24 @@ contract Auctioneer is CloneFactory {
 
     /// @notice Tears down an open auction with given address immediately.
     /// @dev Can be called by contract owner to early close an auction if it
-    ///      is no longer needed.
-    function earlyCloseAuction(Auction auction) internal {
+    ///      is no longer needed. Bear in mind that funds from the early closed
+    ///      auction last on the auctioneer contract. Calling code should take
+    ///      care of them.
+    /// @return Amount of funds transferred to the auction at the time of closing.
```

Comment:
More precisely, isn't it "the amount transferred to this contract by the Auction being early closed"?

Reply:
https://github.com/keep-network/coverage-pools/pull/43/commits/495e090974a40c594c8304435a477d3c5eab2fe6